datadog_checks_base/tests/test_wmisampler.py

```diff
@@ -55,3 +47,22 @@ def test_format_filter_tuple():
     sampler = WMISampler(logger=None, class_name='MyClass', property_names='my.prop', filters=filters)
     formatted_filters = sampler.formatted_filters
     assert formatted_filters == " WHERE ( a < '3' )"
+
+
+@requires_windows
+@pytest.mark.unit
+def test_format_filter_win32_log():
+    query = collections.OrderedDict(
+        {
+            'TimeGenerated': ('>=', '202056101355.000000+'),
+            'Type': [('=', 'Warning'), ('=', 'Error')],
+            'SourceName': [('=', 'MSSQLSERVER')],
+        }
```

Comment:
```suggestion
        (
            ('TimeGenerated', ('>=', '202056101355.000000+')),
            ('Type', [('=', 'Warning'), ('=', 'Error')]),
            ('SourceName', [('=', 'MSSQLSERVER')]),
        )
```

Reply:
but I want to specifically test what win32 is using, which is a list of dict

Reply:
On Python 2, passing a dict (which is unordered) is the same as not using ordereddict. You need to pass sequence of 2-tuples. See https://stackoverflow.com/questions/15733558/python-ordereddict-not-keeping-element-order